doc/docker/install.yml

```diff
@@ -45,7 +45,7 @@ services:
       for i in $$(seq 1 3); do composer install --no-progress --no-interaction --prefer-dist --no-suggest --optimize-autoloader && s=0 && break || s=$$? && sleep 1; done; (exit $$s);
       mkdir -p web/var;
       php /scripts/wait_for_db.php;
-      php bin/console ezplatform:install ${INSTALL_EZ_INSTALL_TYPE};
+      composer ezcommerce-install
```

Comment:
This and platform.sh change for install should be changed to use `composer ezplatform-install` upstream, we should not hard code such things in docker and platform.sh config.